src/plugins/intel_cpu/tests/functional/behavior/export_import.cpp

```diff
@@ -6,6 +6,7 @@
 #include "openvino/runtime/compiled_model.hpp"
 #include "common_test_utils/test_common.hpp"
 #include "ov_models/builders.hpp"
```

Comment:
please, remove `ov_models/builders.hpp` where possible. Further it will be harder to do

Reply:
done